write broker functions to add, claim, and remove, and unit tests

patriacaelum opened the pull request.

LinuxIsCool commented:
Things to consider:

* Proposal Inverter Parameters can have doc strings
* 'Required stake' should perhaps be named 'Minimum Stake'. If so, then update in all places including the spec. 
* Rename 'b' to 'broker' 
* Add doc string to the proposal inverter class.

patriacaelum commented:
All of the proposed changes have been implemented. In particular:
- Added docstrings to the Proposal Inverter class and other methods in the class
- Renamed `b` to `broker`
- Renamed 'Required Stake' to 'Minimum Stake' in the documentation, readme, and hackmd
- Created a `get_horizon()` method to determine how many epochs there is enough funds for left
